.gitversion

```diff
@@ -1,3 +1,3 @@
-version = 0.1.1
+version = 0.1.2
```

Comment:
Wouldn't this cause the resolution system to consider it a breaking change (i.e. any other package in an installation that require 0.1.x would cause image resolution to fail)

Reply:
No, 0.2 is compatible with 0.1. Only if somebody specifically asks for 0.1 then this would fail.